addons/udes_stock/views/stock_inventory.xml

```diff
@@ -16,5 +16,33 @@
                 </xpath>
             </field>
         </record>
+
+        <record id="view_inventory_form" model="ir.ui.view">
+            <field name="name">stock.inventory.form</field>
+            <field name="model">stock.inventory</field>
+            <field name="inherit_id" ref="stock.view_inventory_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//header/button[@name='action_done']" position="attributes">
+                    <attribute name="name">button_done</attribute>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="view_adjust_reserved" model="ir.ui.view">
+            <field name="name">stock.inventory.adjust.reserved.view.form</field>
+            <field name="model">stock.inventory</field>
+            <field name="priority">20</field>
+            <field name="arch" type="xml">
+                <form string="Adjust Reserved Quants?">
```

Comment:
Do we want to call them reserved quants or reserved stock? Might be more user friendly.

Reply:
Good idea, I'll change this